pylint/config.py

```diff
@@ -526,6 +527,28 @@ def parse(self, to_parse, config):
         pass
 
 
+def make_commenter(description):
+    """Make a function to add comments from re.sub."""
+    def make_comment(match):
+        """Make the replacement string including commented desciption."""
+        comment = '\n'.join(
+            '# {}'.format(line) for line in textwrap.wrap(description))
+        return match.expand(r'{}\n\g<0>'.format(comment))
```

Comment:
Fun fact, `\0` doesn't seem to work with expand, even when using a raw string. It gets turned into a null byte, not the whole match.